src/middlewares/aggregate.js

```diff
@@ -4,7 +4,7 @@ const { MEDIATOR } = require('../../config');
 const {logger} =require('../utils/logger');
 
 const cronService = () => cron.schedule('0 0 15 * *', () => {
```

Comment:
This schedule is too aggressive especially with increasing data ingest.

We also need to rethink this (integration) in light of our production deployment being multi-instance

Reply:
Current schedule is mid of every month.